mesonbuild/coredata.py

```diff
@@ -970,9 +970,9 @@ def write_cmd_line_file(build_dir: str, options: argparse.Namespace) -> None:
 
     properties = OrderedDict()
     if options.cross_file:
-        properties['cross_file'] = options.cross_file
+        properties['cross_file'] = list(map(os.path.abspath, options.cross_file))
```

Comment:
please do not use `list` and `map`, they're semi-deprecated, a list comprehension is much easier to read and often 2-3x faster:
```python
[os.path.abspath(f) for f in options.cross_file]
```